development_environment

```diff
@@ -127,5 +127,8 @@ FLOWETL_POSTGRES_USER=flowetl
 FLOWETL_POSTGRES_HOST=flowetl_db
 FLOWETL_POSTGRES_DB=flowetl
 
+# Need this to install airflow locally
+SLUGIFY_USES_TEXT_UNIDECODE=yes
```

Comment:
Out of interest, what does this actually do?

Reply:
It's acceptence of some licence...

Reply:
Can't install airflow without it... https://stackoverflow.com/a/52210596

Or distribute anything that uses it...

Reply:
Looks like it's not actually required any more: https://github.com/apache/airflow/blob/master/UPDATING.md#airflow-1103

Reply:
Not sure that's true but will give it a go since now using the newer version but I'm sure I got an error...

Reply:
Waheyyyyyyy! FYI didn't doubt you James just Airflow...